### DISCUSSION TITLE
The "fade right" or "fade left" transition doesn't work on the element when the modal (the elements container) is hidden

### DISCUSSION THREAD

**Discussion by mohammadhossein1976** [fomantic/Fomantic-UI#1901] [Behaviors (Api, Form Validation, Visibility)] [answered]
hi, how are you? i want to know why fomantic's "fade right" and "fade left" and also many other transitions don't work on the element, after when it's container (example a modal) is hidden.

thank you

**Comment by lubber-de** [comment, marked as answer]
if the container, as the parent element, is hidden, then also the childs are hidden,  because they inherit their visibility.
